vanilla-tests/src/main/java/fixtures/bodycomplex/models/Goblinshark.java

```diff
@@ -41,7 +49,15 @@ public final class Goblinshark extends Shark {
     public Goblinshark(@JsonProperty(value = "length", required = true) float length,
         @JsonProperty(value = "birthday", required = true) OffsetDateTime birthday) {
         super(length, birthday);
-        this.fishtype = "goblin";
+    }
+
+    /**
+     * Get the fishtype property: The fishtype property.
+     * 
+     * @return the fishtype value.
+     */
+    public String getFishtype() {
```

Comment:
There should be an @Override? Seems it overrides parent discriminator getter.

Reply:
This is fixed now